flormcalvo/poc-csyd-mentor
# Dev 002

## Rationale
Create Navbar and fix up page margin
- Search box
- logo
- mobile view

## Advice for Reviewers & Testing Notes
- Run the page locally
- Please see if it work and the color is ok
- The Navbar is fixed at the top.
- no margin
- check the folder structure is correct.

## Screenshots:

**Mobile view:**
![image](https://user-images.githubusercontent.com/47847951/79323404-83e31f80-7f51-11ea-83ca-ed973ab9d3f9.png)

**Desktop view:**
![image](https://user-images.githubusercontent.com/47847951/79323507-a117ee00-7f51-11ea-8bfb-4e07d135fbe6.png)



## Task Name

-  DEV-002 Navbar UI


## Linting Checklist
